src/cclib/parser/orcaparser.py

```diff
@@ -934,6 +935,229 @@ def energy_intensity(line):
+            # Point group                       ... C2
+            # Used point group                  ... C2
+            # Number of irreps                  ... 2
+            #    Irrep    A has   10 SALCs (ofs=   0) #(closed)=   0 #(active)=   2
+            #    Irrep    B has   10 SALCs (ofs=  10) #(closed)=   0 #(active)=   2
+            #  Symmetries of active orbitals:
+            #    MO =    0  IRREP= 0 (A)
+            #    MO =    1  IRREP= 1 (B)
+            vals = next(inputfile).split()
+            if vals[0] == 'Symmetry':
+                # Symmetry section is only printed if symmetry is used
+                assert vals[-1] == 'ON'
+                point_group = next(inputfile).split()[-1]
+                used_point_group = next(inputfile).split()[-1]
+                num_irreps = int(next(inputfile).split()[-1])
+                num_active = 0
+                for i, line in zip(range(num_irreps), inputfile):
+                    reg = r'Irrep\s+(\w+) has\s+(\d+) SALCs \(ofs=\s*(\d+)\) #\(closed\)=\s*(\d+) #\(active\)=\s*(\d+)'
+                    groups = re.search(reg, line).groups()
+                    irrep = groups[0]
+                    salcs, ofs, closed, active = map(int, groups[1:])
+                    num_active += active
+                self.skip_line(inputfile, 'Symmetries')
+                for i, line in zip(range(num_active), inputfile):
+                    reg = r'(\d+)  IRREP= (\d+) \((\w+)\)'
+                    groups = re.search(reg, line).groups()
+                    mo, irrep_idx, irrep = groups
+
+            line = next(inputfile)
+            while line != 'SYSTEM-SPECIFIC SETTINGS:\n':
+                line = next(inputfile)
+
+            # SYSTEM-SPECIFIC SETTINGS:
+            # Number of active electrons          ...    4
+            # Number of active orbitals           ...    4
+            # Total number of electrons           ...    4
+            # Total number of orbitals            ...   20
+
+            num_el = int(next(inputfile).split()[-1])
+            num_orbs = int(next(inputfile).split()[-1])
+            total_el = int(next(inputfile).split()[-1])
+            total_orbs = int(next(inputfile).split()[-1])
+
+            self.skip_lines(inputfile, ['b', 'Determined'])
+
+            # Determined orbital ranges:
+            #    Internal       0 -   -1 (   0 orbitals)
+            #    Active         0 -    3 (   4 orbitals)
+            #    External       4 -   19 (  16 orbitals)
+            orbital_ranges = []
+            for i in range(3):
+                vals = next(inputfile).split()
+                start, end, num = int(vals[1]), int(vals[3]), int(vals[5])
+                # Change from inclusive to exclusive in order to match python
+                end = end + 1
+                assert end - start == num
+                orbital_ranges.append((start, end, num))
+
+            line = next(inputfile)
+            while line != 'CI-STEP:\n':
```

Comment:
Robust in windows? Maybe you can do something like `while line[:8] != 'CI-STEP:'`.

Reply:
Or l.strip() != "..."